fda-model/alternativeComparisonReport/AlternativeComparisonReport.cs

```diff
@@ -26,7 +26,7 @@ public static AlternativeComparisonReportResults ComputeDistributionOfAAEQDamage
             {
                 AlternativeResults damageReducedOneAlternative = new AlternativeResults(withProjectAlternativeResults.AlternativeID);
 
-                foreach (ConsequenceResult withProjectDamageResult in withProjectAlternativeResults.ConsequenceResults.ConsequenceResultList)
+                foreach (ConsequenceDistributionResult withProjectDamageResult in withProjectAlternativeResults.ConsequenceResults.ConsequenceResultList)
                 {
                     //ConsequenceResults withoutProjectDamageResults = withoutProjectAlternativeResults.GetConsequenceResults(withProjectDamageResults.RegionID);
```

Comment:
There's a ton of commented out code here that's got me confused. What's happening?

Reply:
forgot to remove after refactor. nuked!!